# Generated by Django 5.2.4 on 2025-07-05 15:54

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0014_alter_results_infection_detected'),
    ]

    operations = [
        migrations.AddField(
            model_name='customuser',
            name='security_answer_1',
            field=models.CharField(blank=True, help_text='Answer to the first security question', max_length=255, verbose_name='Security Answer 1'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='security_answer_2',
            field=models.CharField(blank=True, help_text='Answer to the second security question', max_length=255, verbose_name='Security Answer 2'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='security_answer_3',
            field=models.CharField(blank=True, help_text='Answer to the third security question', max_length=255, verbose_name='Security Answer 3'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='security_question_1',
            field=models.CharField(blank=True, help_text='First security question for account recovery', max_length=255, verbose_name='Security Question 1'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='security_question_2',
            field=models.CharField(blank=True, help_text='Second security question for account recovery', max_length=255, verbose_name='Security Question 2'),
        ),
        migrations.AddField(
            model_name='customuser',
            name='security_question_3',
            field=models.CharField(blank=True, help_text='Third security question for account recovery', max_length=255, verbose_name='Security Question 3'),
        ),
    ]
